From b9680e868d4bdf9337e542b08a559e9ca5ea6cc0 Mon Sep 17 00:00:00 2001
From: Paulo Alcantara <pc@manguebit.org>
Date: Tue, 25 Nov 2025 17:54:04 -0300
Subject: [PATCH 2/2] docs: update echo_interval description

It is '3 * echo_interval' since upstream commit f2caf901c1b7 ("cifs:
Fix a race condition with cifs_echo_request").

Reported-by: Alexandros Panagiotou <apanagio@redhat.com>
Signed-off-by: Paulo Alcantara (Red Hat) <pc@manguebit.org>
Cc: Steve French <smfrench@gmail.com>
Cc: linux-cifs@vger.kernel.org
---
 mount.cifs.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mount.cifs.rst b/mount.cifs.rst
index d4890706a0fe..4b6d47447c0e 100644
--- a/mount.cifs.rst
+++ b/mount.cifs.rst
@@ -483,8 +483,8 @@ echo_interval=n
   sets the interval at which echo requests are sent to the server on an
   idling connection. This setting also affects the time required for a
   connection to an unresponsive server to timeout. Here n is the echo
-  interval in seconds. The reconnection happens at twice the value of the
-  echo_interval set for an unresponsive server.
+  interval in seconds. The reconnection happens at three times the
+  value of the echo_interval set for an unresponsive server.
   If this option is not given then the default value of 60 seconds is used.
   The minimum tunable value is 1 second and maximum can go up to 600 seconds.
 
-- 
2.51.1

